prow/config.yaml

```diff
@@ -75,7 +75,7 @@ deck:
 
   tide_update_period: 1s
   rerun_auth_configs:
-    istio:
+    '*':
```

Comment:
Does this allow the istio/maintainers team to rerun istio-private jobs or is only the most specific rerun_auth_config respected?